docs/features/datastores/2_Using Datastores/7_high-performance-datastores.md

```diff
@@ -0,0 +1,84 @@
+
+You can do this via the AWS webui or the CLI:
+
+1. If you’re using the AWS webui head to the S3 service at https://s3.console.aws.amazon.com/s3/buckets, select the desired bucket, click on the permissions tab and select "Edit" on the bucket policy field to add the new policy.
+
+2. If you’re using the AWS CLI, simply create a file containing this policy and then use the following command: 
+
+`aws s3api put-bucket-policy --bucket <YOUR_BUCKET_NAME> --policy file://policy.json`
+
+The policy to add should be the following:
+```
+{
+  "Statement": [{
+    "Sid": "localdev-access-permissions",
+    "Effect": "Allow",
+    "Principal": {
+      "AWS": "arn:aws:iam::<YOUR_AWS_ACCOUNT_ID>:role/grid-s3-access-<YOUR_CLUSTER_NAME>-fsx-global-role"
+    },
+    "Action": [
+      "s3:AbortMultipartUpload",
+      "s3:DeleteObject",
+      "s3:PutObject",
+      "s3:Get*",
+      "s3:List*",
+      "s3:PutBucketNotification",
+      "s3:PutBucketPolicy",
+      "s3:DeleteBucketPolicy"
+    ],
+    "Resource": [
+      "arn:aws:s3:::<YOUR_BUCKET_NAME>",
+      "arn:aws:s3:::<YOUR_BUCKET_NAME>/*"
+    ]
+  }]
+}
+```
+
+# Creating High-Performance Datastores
+
+After configuring access you can create a high performance datastore by passing the `--hpds` flag to the CLI when executing the `grid datastore create` command. 
+
+For example: `grid datastore create s3://my-s3-bucket/ --cluster my-byoc-cluster --hpds`
+
+This will create the datastore using the default capacity of 1200GiB and the default throughput of 125MB/s/TiB and after some time (Between 15 minutes and 2 hours depending on the number of files in your S3 bucket) the datastore will be available to use. You can monitor the datastore state normally by using the grid datastore command.
+
+If your dataset is larger than 1200GiB you can use the `--hpds-capacity` flag to specify a higher capacity up to 64,800GiB. Note that capacity must be 1200, 2400 or a multiple of 2400.
+
+If you’re going to be using your datastore with a high number (>20) of experiments or sessions at the same time you can also specify a higher throughput parameter by using the `--hpds-throughput` flag. You may choose between the low (125MB/s/TiB), medium (500MB/s/TiB) or high (1000MB/s/TiB) options.
+
+Do note that on single-instance usage (For example for a single experiment or session) the higher throughput options have heavily diminishing returns (Especially as these datastores can burst to speeds higher than their nominal throughput for sustained periods) and we advise using the low option in this use case in order to reduce costs. 
+
+Finally it's worth noting that there is a period of time after the datastore becomes available to attach and use in sessions during which you will observe reduced performance - this is because the files are being preloaded to the filesystem to ensure consistent performance. This process may take a few hours for particularly high volume datastores.
```

Comment:
```suggestion
Finally, after the Datastore becomes available to attach and use in sessions, you may observe a period of reduced performance. This is because the files are being preloaded to the filesystem to ensure consistent performance. This process may take a few hours for particularly high volume datastores.
```